readme.md

```diff
@@ -231,6 +231,15 @@ Wait for the specified number of milliseconds.
 ```
 Hide the elements matched by the selectors. `method` defines how elements are hidden: "display" sets `display: none`, "opacity" sets `opacity: 0`. Method is "display" by default. Note that only a single string CSS selector is supported here, not an array.
 
+### Cookie match
+```javascript
+{
+  "cookieMatch": "substring",
```

Comment:
`Match` implies that this is a regex, but it is not. For checking for a substring we should use appropriate naming, e.g. action: `cookieContainsSubstring`. Alternatively, we make this a regex and make that explicit: `cookieRegex`.

Reply:
I was reluctant to add new regex support as it doesn't feel justified right now. My thinking was that we can expand this rule in the future if necessary, by adding new parameters, something like:
```json
{
  "cookieMatch": "^some-regex$",
  "regex": true"
}
```

Naming is hard, do you have ideas on the name that would not be confusing, but still potentially fit regex scenario?

Reply:
I would create a separate action name if we wanted to add regex matching. For string matching, lets use:

```
{
  "cookieContains": "string"
}
```
For regex:
```
  "cookiePattern": "regex"
}
```
(Using "pattern" to because we already have `urlPattern` in the schema).

Also, `reverse` is confusing to me. I think it would be better to define a generic negation rule or operator. Either:
```
{
  "not": RuleStep
}
```
or as an annotation that can be put on any `RuleStep`:
```
{
  "exists": ".button",
  "negated": true
}
```

Reply:
Agreed, this makes sense. For the negation, I prefer the annotation style. Will make the change shortly.